packages/react/src/components/StorybookSnapshots.test.js

```diff
@@ -87,6 +87,10 @@ describe(`Storybook Snapshot tests and console checks`, () => {
     // https://stackoverflow.com/questions/53271193/typeerror-scrollintoview-is-not-a-function
     window.HTMLElement.prototype.scrollIntoView = jest.fn();
     window.HTMLElement.prototype.scrollTo = jest.fn();
+    window.getComputedStyle = () => ({
+      getPropertyValue: () => '25',
+    });
+    window.document = new Document();
```

Comment:
We should reset these after the tests.

Reply:
We should be good to go once this is addressed.

Reply:
thanks for catching, this is resolved now @davidicus @kevinsperrine